api/src/lists/lists.service.ts

```diff
@@ -90,8 +92,12 @@ export class ListsService {
     return userListDetails;
   }
 
-  async createList(createList: CreateListDto, userId: string) {
-    return await this.listsDao.createList(createList, userId);
+  async createList(movies: CreateListDto, userId: string) {
```

Comment:
Why is the param called `movies`? isn't it a list?

Reply:
no it's the body of the request consisting of a list name and a movie array. I'll update accordingly. A lot of this will change with the new `movielist` module I think thankfully.